spec/models/miq_policy_spec.rb

```diff
@@ -56,6 +56,13 @@
       end
     end
 
+    describe "#display_name" do
+      it "renders a combination of towhat, mode, and description" do
+        expected = "VM and Instance Control: #{policy.description}"
+        expect(policy.display_name).to eq(expected)
```

Comment:
Note:  This test was written with the previous implementation of `#display_name`

```ruby
  def display_name
    @display_name ||= "#{Dictionary.ui_lookup(:model => towhat)} #{mode.capitalize}: #{description}"
  end
```

As such, it seems like this should be a proper replacement for the code here:

https://github.com/ManageIQ/manageiq-ui-classic/blob/60abc4f0abfdb631d57bcfe7b1e299e2d6e337b0/app/controllers/miq_policy_set_controller.rb#L132